### PR TITLE
Added new database table to user uploaded images

### PR DESCRIPTION
**Pull request by guavajellyaaron**
Let's have a quick conversation about this pull request before you merge it.

Added new database table to user uploaded images
Added image upload via ajax to the create and update strategy pages
Added generic handler (ashx) file to handle image

### CODE REVIEW

File: projects/BossCheatSheets.Database/dbo/Tables/Strategy_UserImage.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[Strategy_UserImage] (
+    [ImageID]            INT             IDENTITY (1, 1) NOT NULL,
+    [UserID]             INT             NOT NULL,
```
Comment (happygaijin) [inline projects/BossCheatSheets.Database/dbo/Tables/Strategy_UserImage.sql]: Should this join on user or strategy? These images are per strat yeah?

**Comment by happygaijin** [comment]
Other than switching the image to join on strat, this is good.

**Comment by guavajellyaaron** [comment]
If the strategy hasn't been created we wont have the ID. how should I handle this?

**Comment by happygaijin** [comment]
EF should see that the image is a child of the strat and insert the strat first and then the image accordingly. If EF isn't capable of doing this, then we need to do that ourselves.

**Comment by happygaijin** [comment]
I'm a little worried it might be tough to back out of this user image relationship. But I'll merge anyway so we can move forward and we'll cross that bridge when we get to it

**Pull request merged**